projects/archiver/src/tasks/notification/helpers/device-notifications.ts

```diff
@@ -40,31 +50,73 @@ const scheduleDeviceNotification = async (
     return { statusCode: status, statusText }
 }
 
-export const scheduleDeviceNotificationIfInFuture = async (
-    issueData: IssueNotificationData,
-    cfg: { domain: string; apiKey: string },
-) => {
-    const scheduleTime = createScheduleTime(issueData.issueDate)
+export type DeviceNotificationStatus = 'skipped' | 'send'
```

Comment:
maybe `toSend` instead of `send` as send feels like a command and not a status. (or the opposite, `skip` instead of `skipped` maybe?)

Reply:
for schedule notification status maybe 'scheduled' and 'skipped'  ?

Reply:
I wonder if we should change the handler such that instead of passing the success status in we allow the block to return a tuple of `[ newStatus, output ]` and update the status based on that.